clients/cpp-console/include/AudioPlayer.h

```diff
@@ -11,15 +12,14 @@
 /// </remarks>
 class IAudioPlayer{
     public:
-    
         /// <summary>
         /// AudioPlayerFormat is an enum that can be used in the AudioPlayer
-        /// implementation. It is passed in as a parameter to the Open and Play functions.
+        /// implementation. It is passed in as a parameter to the Initialize and Play functions.
         /// </summary>
         /// <example>
```

Comment:
You can remove the <example> here... the example should be on the Initialize method

Reply:
removed